setup.cfg

```diff
@@ -49,7 +48,8 @@ install_requires =
     sphinx>=2,<4
     sphinx-comments
     sphinx-copybutton
-    sphinx-external-toc~=0.2.0
+    sphinx-external-toc~=0.1.0
```

Comment:
@mmcky it looks like we have now _downgraded_ sphinx-external-toc...is that intentional?

Reply:
most definitely it should not be

Reply:
@choldgraf I merged #1364
Not sure how that happened as I had merged in the latest `master` before merging. Hmm ...

Reply:
thanks for picking that up